src/lib/util/address/address.ts

```diff
@@ -0,0 +1,3 @@
+// Address validation
+export const isValidEthAddress = (text: string) =>
+	/^0x[a-fA-F0-9]{40}$/.test(String(text).toLowerCase());
```

Comment:
I didn't realise, but looks like ethers has a function for this:
https://docs.ethers.io/v5/api/utils/address/#utils-isAddress

Reply:
Oh sick! this is nice. Thanks. updated